### ISSUE TITLE
Update Question, RM2 update from manufacture first?

### ISSUE THREAD

**Issue by Bricewilliamsinc** [ddvk/remarkable-hacks#272]
I keep auto update off, can I update ddvk hack to newest version without having to get the RM2 manufactures 2.6 update first?

**Comment by ddvk** [comment]
no, the patch is version specific and needs the original xochitl, so you need to update first
